x-pack/plugin/eql/src/test/java/org/elasticsearch/xpack/eql/planner/QueryFolderFailTests.java

```diff
@@ -62,51 +62,78 @@ public void testBetweenWrongTypeParams() {
                 error("process where between(process_name, \"s\", \"e\", false, 2)"));
     }
 
-    public void testCIDRMatchNonIPField() {
+    public void testCIDRMatchAgainstField() {
```

Comment:
I rearranged these test methods alphabetically in hopes that it makes git conflicts less likely